### ISSUE TITLE
A11y_code.visualstudio.com_APIGetStarted _YourFirstExtension_ScreenReader: Same name(label) or ambiguous name is provided for all the video content.

### ISSUE THREAD

**Issue by Mohini78** [microsoft/vscode-docs#2783]
“[Check out Accessibility Insights](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com)
Windows Version: Windows10
Browser Version:
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : Narrator, NVDA2019.1.1
MAS Violated: MAS 1.3.1, MAS4.1.2

**Repro Steps:**
Navigate to the URL: [https://code.visualstudio.com/api/get-started/your-first-extension](https://code.visualstudio.com/api/get-started/your-first-extension) OR
follow below steps:
1.Launch the application URL: [https://code.visualstudio.com](https://code.visualstudio.com)
2.Navigate to "API" link.
3.From left navigation, select "Get Started".
4.Navigate to "Your First Extension" link.
5.Turn on the narrator using Ctrl + Win + Enter.
6.Navigate to the videos present on the page using TAB.

**Actual:**
Same name(label) or ambiguous name is provided for all the video content i.e. Video group, hence same is announced by the narrator.

This issue is observed in Edge with Narrator and Chrome with NVDA. 

**Expected:**
Different and descriptive name(label) must be provided for all the video content.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Screen reader only users will get confused while navigating to different videos having different content if same name is provided to all the videos.

**MAS Reference:**
MAS 1.3.1 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

MAS4.1.2 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Attachment for Reference:**
![A11y_code visualstudio com_APIGetStarted _YourFirstExtension_1](https://user-images.githubusercontent.com/51443731/59920099-10fb3c00-9447-11e9-821d-acb18c0cad0d.png)
![A11y_code visualstudio com_APIGetStarted _YourFirstExtension_2](https://user-images.githubusercontent.com/51443731/59920100-10fb3c00-9447-11e9-8592-09ddcc0fd2d8.png)
![A11y_code visualstudio com_APIGetStarted _YourFirstExtension_3](https://user-images.githubusercontent.com/51443731/59920101-1193d280-9447-11e9-85d6-5269e91a69be.png)
